src/numGenerator.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const numGenerator = () => {
+  let digits = '';
+
+  for (let i = 0; i < 4; i++) {
+    const nextDigt = Math.round(Math.random() * 9);
+
+    if (!digits.includes(nextDigt)) {
+      digits += nextDigt;
+    } else {
+      i--;
+    }
+  }
```

Comment:
In the worst-case scenario, we could spend eternity to generate a number with unique digits because the random number could keep generating numbers that have already been included. Statistically it is rather unlikely, but still this is an area to improve the efficiency, especially in case of scaling the app.

Although it will work, so 🆗

Reply:
Probability that finding last digit for longer than 0.2s is almost equal 0, so tere is no option to run this infinitely.
Random number was often used as a key while mapping array in React. Theoretically all keys could be the same, but probability is almost 0, so it's used.
Task is already accepted by mentor.

Reply:
yeah, sure, definitely this is all fine to keep it this way, just wanted to point it out :)